### PR TITLE
[JavaScript] Add 1-b and 2-b exercises

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

**Comment by pixeleet** [comment]
@ErikSchierboom  LGTM
Like a lot that for unfound values you just return null, this gives way for the exercise, to be later extended by a `default` property to fallback on if the path is not found, if you choose so.

Thank you for finishing the work instead of me :)

File: exercises/javascript-1-b/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+- an `object` (the `Object` which data to extract),
+- a `path` (a `String` with the data's path, separated by dot characters)
+
+The function should then return the value within the object that matches the path:
+
+```javascript
+const team = { coach: { name: "Jane" }, name: "Hoop Masters" };
+
+extract(input, "name");
```
Comment (mmmmmrob) [inline exercises/javascript-1-b/README.md]: should these be `extract(team, "name")`?

File: exercises/javascript-2-b/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+- an `object` (the `Object` which data to extract),
+- a `path` (a `String` with the data's path, separated by dot characters)
+
+The function returns the value within the object that matches the path, returning `null` if no value was found:
+
+```javascript
+const team = { coach: { name: "Jane" }, name: "Hoop Masters" };
+
+extract(input, "name");
```
Comment (mmmmmrob) [inline exercises/javascript-2-b/README.md]: `extract(team, "name");`

File: exercises/javascript-2-b/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+// => "Hoop Masters"
+
+extract(input, "coach.name");
```
Comment (mmmmmrob) [inline exercises/javascript-2-b/README.md]: `extract(team, "coach.name");`

File: exercises/javascript-2-b/README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      position: "Point guard"
+    },
+    {
+      name: "Maya",
+      position: "Center"
+    }
+  ]
+};
+
+extract(input, "players[0].name");
```
Comment (mmmmmrob) [inline exercises/javascript-2-b/README.md]: `extract(team, "players[0].name");`

File: exercises/javascript-2-b/README.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+// => "Becky"
+
+extract(input, "players[1].position");
```
Comment (mmmmmrob) [inline exercises/javascript-2-b/README.md]: `extract(team, "players[1].position");`

**Comment by iHiD** [comment]
@mmmmmrob Thanks for those catches! I tried to re-request review from you, but GH UI seems to be playing up. You happy with this now?

**Comment by mmmmmrob** [comment]
@iHiD @ErikSchierboom Yeah, I only had time to skim through the readme really. If those are fixed and someone else wants to review that's cool.

**Comment by iHiD** [comment]
Merged. Thanks everyone!

**Pull request merged**